Presentation.Web/Controllers/API/V2/Internal/ItSystemUsages/Mapping/IItSystemUsageMigrationResponseMapper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Core.ApplicationServices.Authorization;
+using Core.ApplicationServices.Model.SystemUsage.Migration;
+using Core.DomainModel.ItSystem;
+using Presentation.Web.Models.API.V2.Internal.Response.ItSystemUsage;
+using Presentation.Web.Models.API.V2.Response.Generic.Identity;
+using System.Collections.Generic;
+
+namespace Presentation.Web.Controllers.API.V2.Internal.ItSystemUsages.Mapping
+{
+    public interface IItSystemUsageMigrationResponseMapper
+    {
+        ItSystemUsageMigrationV2ResponseDTO MapMigration(ItSystemUsageMigration entity);
+        IEnumerable<IdentityNamePairWithDeactivatedStatusDTO> MapUnusedSystems(IEnumerable<ItSystem> systems);
```

Comment:
this mapping is really not tied to this context. It could be a generic IEntityWithDeactivatedStatusMapper

```
IEnumerable<IdentityNamePairWithDeactivatedStatusDTO> Map<T>(IEnumerable<T> systems) where T: IHasUuid, IHasName, IHasDisabledStatus;
```